swift_browser_ui_frontend/src/common/conv.js

```diff
@@ -279,24 +279,30 @@ export function tokenize(text, ignoreSmallerThan = 2) {
 export const DEV = process.env.NODE_ENV === "development";
 
 export function sortObjects(objects, sortBy, sortDirection) {
-  sortBy = sortBy === "size" ? "bytes" : sortBy;
+  sortBy = sortBy === "size" ? "bytes"
+    : sortBy === "items" ? "count" : sortBy;
 
   objects.sort((a, b) => {
     let valueA = a[sortBy];
     let valueB = b[sortBy];
 
+
```

Comment:
Minor styling thing, but could we possibly have a tad fewer empty lines here?

Reply:
Yes, redundant empty lines are removed. Thanks!